### PR TITLE
PlainSource output jax

### PR DESCRIPTION
**Pull request by pkra**
Adds a new output jax "PlainSource" which exposes a "plain text" source when possible. 

More precisely, this jax creates the following output:

* TeX or asciimath if either is used as input
* TeX or asciimath if MathML is used as input and includes annotationXML for either of these. If both are found, TeX is preferred.

This also fixes https://github.com/mathjax/MathJax/issues/788

### CODE REVIEW

**Comment by pkra** [comment]
@dpvc I've tried to make it IE8 compatible but I still run into issues with TeX input where I get a Math processing error (but nothing productive from lastError -- just unkown runtime error).

**Comment by dpvc** [comment]
Do you want this to be in 2.6?  I don't remember if we discussed that or not.

**Comment by pkra** [comment]
I wouldn't mind. Haven't got to the tests yet. sorry.

**Comment by dpvc** [comment]
I have made a [pull request](https://github.com/pkra/MathJax/pull/1) to your branch to fix some issues.

File: unpacked/jax/output/PlainSource/jax.js [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    Remove: function(jax) {
+      var span = document.getElementById(jax.inputID + "-Frame");
+      if (span) {
+        if (jax.PlainSource.display) {
+          span = span.parentNode
+        }
+        span.parentNode.removeChild(span);
+      }
+      delete jax.PlainSource;
+    },
```
Comment (dpvc) [inline unpacked/jax/output/PlainSource/jax.js]: BTW, the comma here is probably the thing that was tripping up IE8.  Modern browsers are OK with extra commas like this, but older IE versions choke.

Comment (pkra) [inline unpacked/jax/output/PlainSource/jax.js]: Thanks for those!

**Comment by pkra** [comment]
I merged your PR into my branch.

**Pull request merged**